### DISCUSSION TITLE
VRFConsumerBaseV2Plus not found

### DISCUSSION THREAD

**Discussion by JoshuyPan** [Cyfrin/foundry-full-course-cu#2083] [Q&A] [answered]
Hey guys,
i installed the chainlink-brownie contracts into my project and i also updated the forge.toml file as so:

![Screenshot 2024-07-22 224141](https://github.com/user-attachments/assets/320b6da1-01ff-4e55-a0af-0c434e205122)

but my import stills red:

![Screenshot 2024-07-22 224120](https://github.com/user-attachments/assets/348ed61a-027a-4ff6-a64d-b592dfd5f6a4)

and the file exists:

![Screenshot 2024-07-22 224302](https://github.com/user-attachments/assets/746df02d-796e-4366-ad8d-f471f71ed46f)

what can i try to do to solve this?

**Comment by EngrPips** [comment, marked as answer]
Hello @JoshuyPan, Please hover the red squiggly line and show what it says. Try this path instead `@chainlink/contracts/v0.8/vrf/dev/VRFConsumerBaseV2Plus.sol` let's see if it works.

**Reply by JoshuyPan** [reply]
@EngrPips thank you so much, using this path: `@chainlink/contracts/v0.8/vrf/dev/VRFConsumerBaseV2Plus.sol` worked.
thanks for your time and patience

**Reply by EngrPips** [reply]
You are welcome.
